srcpkgs/ffmpeg/template

```diff
@@ -33,6 +33,22 @@ case "$XBPS_TARGET_MACHINE" in
 	mips*) CFLAGS="-mnan=legacy";;
 esac
 
+_apply_patch() {
+	local args="$1" pname="$(basename $2)"
+
+	if [ ! -f ".${pname}_done" ]; then
+		patch -N $args -i $2
+		touch .${pname}_done
+	fi
+}
```

Comment:
Could we make this an exported function to be used in the case of conditional patches?

Reply:
We could but I think conditional patches are a terrible idea and we should not make it more convenient to add them.

Reply:
Fair enough.